repo/fsrepo/fsrepo.go

```diff
@@ -562,7 +562,9 @@ func (r *FSRepo) GetStorageUsage() (uint64, error) {
 
 	var du uint64
 	err = filepath.Walk(pth, func(p string, f os.FileInfo, err error) error {
-		du += uint64(f.Size())
+		if f != nil {
```

Comment:
There is already https://github.com/ipfs/go-ipfs/blob/eb9d0729280669b1e277b082cc28c45c87038020/core/corerepo/gc.go#L147. I added the log, however.